config/locales/views/layouts/en.yml

```diff
@@ -7,5 +7,5 @@ en:
     proposition_header:
       menu: "Menu"
     application:
-      page_title: "Find government data"
-      global_header_text: "Find government data"
+      page_title: "Data.gov.uk - Find Data"
```

Comment:
Is this the official title now?

Reply:
That was a change implemented on master by Toby. I will check in with Gurinder and update if necessary on a different PR